cypress/e2e/CMS/$CreateAnImage/index.js

```diff
@@ -0,0 +1,162 @@
+})
+
+And('user clicks on {string} link in the {string} text area', (title, cartOption) => {
+    cy.get(`div.paragraph-type-title:contains('${cartOption}')`).parent().parent().find(`span:contains('${title}')`).parent().click()
+})
+
+And('user clicks on {string} button to select item', (title) => {
+    cy.getIframeBody('iframe[name="entity_browser_iframe_cgov_content_browser"]').find(`input[value="${title}"]`).click({ force: true })
+})
+
+And('user clicks on {string} link in the Internal Link text area within List Items', (linkBtn) => {
+    cy.get(`tbody summary[role='button'] span:contains('${linkBtn}')`).parent().click({ force: true });
+})
+
+
+And('user selects Display from {string} dropdown', (dropDown) => {
+        cy.get('div#edit-field-display-enlarge-wrapper label').should('contain.text',dropDown);
+        cy.get('select#edit-field-display-enlarge').select("Display");
+})
+
+And('user selects {int} item from the media list', (num) => {
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find('input.form-checkbox').eq(num - 1).check();
+});
+
+And('user selects {string} from the list of media', (image) => {
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find("input#edit-name").type(image);
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find("#edit-submit-cgov-media-browser").click();
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').click().find("input[name*='entity_browser_select'][class='form-checkbox']").check();
+    });
+
+And('user clicks on {string} button to select media', (media) => {
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find(`input#edit-submit[value="${media}"]`).click({ force: true })
+})
+
+And('user selects {string} from Contents dropdown', (selectOption) => {
+    cy.get(`.placeholder:contains("Contents")`).parent().find(`input[value="${selectOption}"]`).click({ force: true });
+    
+})
+
+Then('lead image is displayed with source {string}', (image) => {
+    cy.get('.centered-element img').should('have.attr','src').then((attrSrc) => {
+        expect(attrSrc.replace('.svg', '')).to.include(image.replace('.svg', ''));
+    })
+});
+
+And('caption reads {string}', (caption) => {
+    cy.get('.caption-container p').should('include.text',caption);
+});
+
+And('credit reads {string}', (credit) => {
+    cy.get('.image-photo-credit').should('include.text',credit);
+});
+
+And('{string} button is displayed', (enlarge) => {
+    cy.get('.article-image-enlarge').should('have.text', enlarge).should('be.visible');
+});
+
+Then('feature card image is displaying {string}', (image) => {
+    cy.get('.feature-primary.flex-columns.row img').should('have.attr','src').then((attrSrc) => {
+        expect(attrSrc.replace('.svg', '')).to.include(image.replace('.svg', ''));
+    })
+});
+
+Then('1 item in the list has an image {string}', (image) => {
+    cy.get('.image.item-image > img').should('have.attr','src').then((attrSrc) => {
```

Comment:
1 should not be hardcoded - its param type {int}

Reply:
done

Reply:
item param is not used - remove it from both feature step and from step def

Reply:
done